src/pumipic_lb.hpp

```diff
@@ -336,7 +335,7 @@ namespace pumipic {
           }
         }
       };
-      Omega_h::parallel_for(ptcls_per_elem.size(), selectParticles, "selectParticles");
+      Omega_h::parallel_for(ptcls_per_elem.size() - 1, selectParticles, "selectParticles");
```

Comment:
Is the `-1` skipping the last element?  On first read, I don't see logic in the `selectParticles` kernel that handles the last element (i.e., `elm+1`).  There is an offsets array being indexed by element, but I'd expect that to be sized `numElms+1`.

Reply:
I undid this change.